### PR TITLE
[bitnami/redis-cluster] Fix readiness/liveness scripts typo

### PR DESCRIPTION
**Pull request by kvendingoldo**
We have to use:

[ "$?" -eq "124" ] for bin/sh
OR
[[ "$?" == "124" ]] for bin/bash

In the script we're using bin/sh and for that reason I've fixed scripts

### CODE REVIEW

**Comment by kvendingoldo** [comment]
let me rewrite this script a little bit, 10 minutes

**Comment by kvendingoldo** [comment]
@miguelaeh can you check it, pls?

**Comment by kvendingoldo** [comment]
@rafariossaa @juan131 can you check it too, please?

**Comment by rafariossaa** [comment]
@kvendingoldo ,
   Could you take a look to what I indicated ?

**Comment by kvendingoldo** [comment]
@rafariossaa sorry, where?

File: bitnami/redis-cluster/templates/scripts-configmap.yaml [hunk 1]
```diff
@@ -76,25 +76,22 @@ data:
...
 {{- if .Values.tls.enabled }}
         -p $REDIS_TLS_PORT \
         --tls \
-        --cert {{ template "redis-cluster.tlsCert" . }} \
-        --key {{ template "redis-cluster.tlsCertKey" . }} \
         --cacert {{ template "redis-cluster.tlsCACert" . }} \
+        {{- if .Values.tls.authClients }}
+          --cert {{ template "redis-cluster.tlsCert" . }} \
+          --key {{ template "redis-cluster.tlsCertKey" . }} \
+        {{- end }}
```
Comment (migruiz4) [inline bitnami/redis-cluster/templates/scripts-configmap.yaml:94]: Please fix indentation. Args `--cert` and `--key` have two additional whitespaces.

File: bitnami/redis-cluster/templates/scripts-configmap.yaml [hunk 2]
```diff
@@ -53,17 +51,19 @@ data:
...
+        --cacert {{ template "redis-cluster.tlsCACert" . }} \
+        {{- if .Values.tls.authClients }}
           --cert {{ template "redis-cluster.tlsCert" . }} \
           --key {{ template "redis-cluster.tlsCertKey" . }} \
-          --cacert {{ template "redis-cluster.tlsCACert" . }} \
+        {{- end }}
   {{- else }}
-          -p $REDIS_PORT \
+        -p $REDIS_PORT \
   {{- end }}
```
Comment (migruiz4) [inline bitnami/redis-cluster/templates/scripts-configmap.yaml:63]: Please fix indentation, arguments  should be in line with `-h localhost`

File: bitnami/redis-cluster/templates/scripts-configmap.yaml [hunk 3]
```diff
@@ -12,33 +12,31 @@ metadata:
...
 {{- if .Values.tls.enabled }}
         -p $REDIS_TLS_PORT \
         --tls \
-        --cert {{ template "redis-cluster.tlsCert" . }} \
-        --key {{ template "redis-cluster.tlsCertKey" . }} \
         --cacert {{ template "redis-cluster.tlsCACert" . }} \
+        {{- if .Values.tls.authClients }}
+          --cert {{ template "redis-cluster.tlsCert" . }} \
+          --key {{ template "redis-cluster.tlsCertKey" . }} \
+        {{- end }}
```
Comment (migruiz4) [inline bitnami/redis-cluster/templates/scripts-configmap.yaml:33]: Same here, please fix indentation.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @kvendingoldo,

Please take a look at my comments below. Some linting issues plus `set -e` being removed.

In addition, readiness and liveness probes `exec` continue using `sh`.

**Pull request closed without merging**